R/subscript-tables.R

```diff
@@ -0,0 +1,118 @@
+#' @export
+`[.QTable` <- function(x, ..., drop = TRUE) {
+    # Use sys.call as match.call captures unmatched named arguments into ...
+    used.arguments <- names(sys.call())
```

Comment:
Can you throw all this argument checking code into a separate internal function?

Reply:
Not without a lot of pain involving the called arguments and where they appear on the call stack I fear.